prog/analysis.go

```diff
@@ -10,6 +10,9 @@ package prog
 
 import (
 	"fmt"
+	"io"
+
+	"github.com/google/syzkaller/dashboard/dashapi"
```

Comment:
This dependency feels very wrong. Prog is the lowest level abstract data structure manipulation package that does not import anything. Now it imports some web service API.
Please declare asset types in the prog package.

Reply:
You mean some intermediate asset types in `prog` that will be converted to the `dashapi`-related ones in `syz-manager`?

Reply:
yes